conjure-java-core/src/integrationInput/java/com/palantir/product/DialogueEteService.java

```diff
@@ -0,0 +1,1626 @@
+                        MoreExecutors.directExecutor());
+            }
+
+            @Override
+            public ListenableFuture<Optional<StringAliasExample>> optionalQueryExternalImport(
+                    AuthHeader authHeader, Optional<StringAliasExample> query) {
+                Request.Builder _request = Request.builder();
+                _request.putHeaderParams(
+                        "Authorization",
+                        plainSerDe.serializeBearerToken(authHeader.getBearerToken()));
+                query.ifPresent(
+                        v -> {
+                            _request.putQueryParams("query", Objects.toString(v));
+                        });
+                return Futures.transform(
+                        channel.execute(
+                                DialogueEteService.optionalQueryExternalImport, _request.build()),
+                        optionalQueryExternalImportDeserializer::deserialize,
+                        MoreExecutors.directExecutor());
+            }
+
+            @Override
+            public ListenableFuture<Void> noReturn(AuthHeader authHeader) {
+                Request.Builder _request = Request.builder();
+                _request.putHeaderParams(
+                        "Authorization",
+                        plainSerDe.serializeBearerToken(authHeader.getBearerToken()));
+                return Futures.transform(
+                        channel.execute(DialogueEteService.noReturn, _request.build()),
+                        noReturnDeserializer::deserialize,
+                        MoreExecutors.directExecutor());
+            }
+
+            @Override
+            public ListenableFuture<SimpleEnum> enumQuery(
+                    AuthHeader authHeader, SimpleEnum queryParamName) {
+                Request.Builder _request = Request.builder();
+                _request.putHeaderParams(
+                        "Authorization",
+                        plainSerDe.serializeBearerToken(authHeader.getBearerToken()));
+                _request.putQueryParams("queryParamName", Objects.toString(queryParamName));
+                return Futures.transform(
+                        channel.execute(DialogueEteService.enumQuery, _request.build()),
+                        enumQueryDeserializer::deserialize,
+                        MoreExecutors.directExecutor());
+            }
+
+            @Override
+            public ListenableFuture<List<SimpleEnum>> enumListQuery(
+                    AuthHeader authHeader, List<SimpleEnum> queryParamName) {
+                Request.Builder _request = Request.builder();
+                _request.putHeaderParams(
+                        "Authorization",
+                        plainSerDe.serializeBearerToken(authHeader.getBearerToken()));
+                for (SimpleEnum v : queryParamName) {
+                    _request.putQueryParams("queryParamName", Objects.toString(v));
```

Comment:
thoughts on using `v.toString()` instead of `Objects.toString(v)` (or some other validation) as we intentionally don't want people to be able to put nulls onto the wire?

Reply:
using `Objects.toString` allows us to cleanly handle collections of boxed primitives